scripts/firedrake-install

```diff
@@ -717,7 +723,7 @@ def build_and_install_h5py():
     else:
         try:
             petsc_dir = check_output(python +
-                                     ["-c", "import petsc; print petsc.get_petsc_dir()"]).strip()
+                                     ["-c", "import petsc; print(petsc.get_petsc_dir())"]).strip()
```

Comment:
`print` function in Python 3, superfluous parenthesis in Python 2, but works in both.